mmpose/datasets/datasets/top_down/topdown_posetrack18_dataset.py

```diff
@@ -191,6 +201,17 @@ def evaluate(self, outputs, res_folder, metric='mAP', **kwargs):
                 oks_nmsed_kpts[image_id].append(
                     [img_kpts[_keep] for _keep in keep])
 
+            if self.use_nms:
```

Comment:
Seems not related to this PR?

Reply:
Em, this is not related to batch inference. Should I open another proposal for this? Or maybe just merge it.

Reply:
Just leave it here for now.

Reply:
Split independent changes to separate prs next time

Reply:
Em, I'll open another  pr to add **use_nms**.